### PR TITLE
Eswe 44 courses quals detail updates

### PR DESCRIPTION
**Pull request by natclamp-moj**
This PR:
- fixes a bug so that the dates show 'fromDate to toDate' rather than 'toDate to toDate'
- changes the table to display temporarily withdrawn courses as temporarily withdrawn, rather than grouped with ongoing courses
- updates tests to match changes

### CODE REVIEW

File: integration-tests/integration/prisonerProfile/prisonerCoursesAndQualificationsDetails.spec.js [hunk 1]
```diff
@@ -17,12 +17,65 @@ context('Prisoner courses and qualifications details page', () => {
...
+
+    it('should show the correct content and number of results', () => {
+      cy.visit(`/prisoner/${offenderNo}/courses-qualifications`)
+      cy.get('h1').should('have.text', 'John Smith’s courses and qualifications')
+      cy.get('[data-test="courses-quals-result-number"]').should('have.text', 'Showing 1 result')
+      cy.get('[data-test="courses-qualifications-returnLink"]').should('exist')
+      cy.get('tbody tr')
+        .find('td')
+        .then(($cells) => {
+          expect($cells.length).to.eq(6)
```
Comment (JamieDraperUK) [inline integration-tests/integration/prisonerProfile/prisonerCoursesAndQualificationsDetails.spec.js:69]: Don't worry about changing it now. But we normally extract the data into an object. A random example pulled from another test: `
const toOffender = ($cell) => ({
  name: $cell[1]?.textContent,
  prisonNo: $cell[2]?.textContent,
  location: $cell[3]?.textContent,
  alerts: $cell[4]?.textContent,
  cellHistoryText: $cell[5]?.textContent,
  changeCellText: $cell[6]?.textContent,
})`

File: views/prisonerProfile/prisonerWorkAndSkills/prisonerCoursesQualificationsDetails.njk [hunk 2]
```diff
@@ -48,7 +48,7 @@
       <div class="govuk-grid-column-full">
         {{ printLink(align = "right") }}
         <div class="courses-qualifications-pagination-print-alignment">
-          <p class="moj-pagination__results" >Showing <b>{{ coursesDetailsList.length }}</b> {% if coursesDetailsList.length > 1 %}results{% else %}result{% endif %}</p>
+          <p class="moj-pagination__results" data-test='courses-quals-result-number'>Showing <b>{{ coursesDetailsList.length }}</b> {% if coursesDetailsList.length > 1 %}results{% else %}result{% endif %}</p>
```
Comment (JamieDraperUK) [inline views/prisonerProfile/prisonerWorkAndSkills/prisonerCoursesQualificationsDetails.njk:51]: Minor but mixed single and double quotes.

**Review by JamieDraperUK — APPROVED** [review]
LGTM

File: integration-tests/integration/prisonerProfile/prisonerCoursesAndQualificationsDetails.spec.js (hunk 1 above)
Comment (natclamp-moj) [reply integration-tests/integration/prisonerProfile/prisonerCoursesAndQualificationsDetails.spec.js:69]: Oh interesting - hadn't seen that! Thanks

**Pull request merged**